cmd/updater/systemd-setup.sh

```diff
@@ -1,26 +1,48 @@
 #!/bin/bash
+# shellcheck disable=2086,2164
 #
 # Run this script to set up the systemd service for algod.
-# The arguments are the username and groupname that algod
-# should run as.
+# The arguments are the username and groupname that algod should run as.
+# If wanting to run as a user service, substitute "--user" for the groupname.
+
+setup_user() {
+    mkdir -p "/home/$1/.config/systemd/user"
```

Comment:
This may be a better way to get the user home dir in case anyone does any nonstandard setups
```
eval echo ~$1
```

Reply:
https://unix.stackexchange.com/questions/332579/is-it-always-safe-to-use-eval-echo

Reply:
I mean, we could make sure the string is an actual user first if we're worried about us maliciously using our scripts

Reply:
If there's a better way that's fine too, we just need to make sure we're actually using the user's home directory

Reply:
Since this script is specified for bash, we don't need the eval apparently. That part's only necessary on sh, so this should work
```
echo ~"$1"
```

Reply:
Actually never mind, the eval is still necessary I think. It worked on my mac, but on ubuntu I had to use the eval

```
root@22112cf00d84:/# useradd -d /tmp/hodor hodor
root@22112cf00d84:/# echo ~"hodor"
~hodor
root@22112cf00d84:/# eval echo ~"hodor"
/tmp/hodor
```

Reply:
I fixed this.  It's getting it from `/etc/passwd` which is the most reliable way.